UI/window-basic-main.cpp

```diff
@@ -6081,8 +6088,8 @@ void OBSBasic::on_streamButton_clicked()
 		switch (action) {
 		case StreamSettingsAction::ContinueStream:
 			break;
-		case StreamSettingsAction::OpenSettings:
-			on_action_Settings_triggered();
+		case StreamSettingsAction::OpenSettingsStream:
+			on_action_Settings_triggered(1);
```

Comment:
I think this would be a great place for an enum like `ObsSettingsOpen::StreamSettings` in case the menu is rearranged later, intent would be more clear.